doc/widgets/bagofwords.rst

```diff
@@ -24,9 +24,39 @@ A set of words from corpus.
 Description
 -----------
 
-No description yet for this widget.
+**Bag of Words** model creates a table with word counts for each data instance (document). The count can be either absolute, binary (contains or does not contain) or sublinear (logarithm of the term frequency). Bag of words model is required in combination with :doc:`Word Enrichment <wordenrichment>` and any kind of predictive modelling.
```

Comment:
creates a `table` — might be confused with Orange's Table. Maybe we rather use Corpus? Or rephrase a bit?